### DISCUSSION TITLE
In Next.js v13.4.5, I can't push code using command. I used push function of vscode editor to push code.

### DISCUSSION THREAD

**Discussion by cupid20103** [vercel/community#2874] [Help]
### Summary

Hello, Everyone!
I have some issues pushing my code.
This is Husky's pre-push command that runs before pushing my code.

```
#!/usr/bin/env sh
. "$(dirname -- "$0")/_/husky.sh"

yarn build
```

but I get same error like this.

```
stdin is not a tty
husky - pre-push hook exited with code 1 (error)
error: failed to push some refs to 'https://github.com/Gods-of-love/learn-nextjs-v13.git'
```

How to solve it?

### Example

https://github.com/Gods-of-love/learn-nextjs-v13.git

### Steps to Reproduce

![error](https://github.com/vercel/community/assets/102618556/d915fc8c-16f3-45e7-8060-d47a931e126d)

**Comment by amyegan** [comment]
Hi @cupid20103. It sounds like there is a problem connecting to your remote repo from your local environment. The error "stdin is not a tty" make it sound like this is an ssh issue. You should check your connection settings. I found a couple of StackOverflow posts that might help you solve the problem:

- https://stackoverflow.com/questions/71670983/stdin-is-not-a-tty-when-running-git-push-github-over-ssh
- https://stackoverflow.com/questions/12480284/ssh-error-when-executing-a-remote-command-stdin-is-not-a-tty

Please contact GitHub support if you need more help. They'll have more insight into their logs to help you figure out what is going wrong.
